src/components/VFooter/VFooter.vue

```diff
@@ -90,6 +88,10 @@ export default defineComponent({
       required: false,
       default: undefined,
     },
+    languageProps: {
+      type: Object,
```

Comment:
Can you add the proper type annotation to this? `Object` will just resolve this prop's type to `any` in TS. You can see an example for how this is done in `VIconButton`.

Reply:
Thanks for the reference! Implemented in 134800f.